Data Structures/HashMap/Main.java

```diff
@@ -0,0 +1,161 @@
+import java.util.Scanner;
+
+class Node {
```

Comment:
You must moved the single classes into singel files. Each class in one file. Or you write innert classes.